lab-said/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "lab-said",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "build": "webpack",
+    "watch": "webpack-dev-server --inline --hot"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "css-loader": "^0.28.7",
+    "cowsay": "^1.2.1",
```

Comment:
it looks like you installed cowsay instead of cowsay-browser, and webpack will not compile.